metagraph/src/cli/query.cpp

```diff
@@ -484,16 +827,41 @@ int query_graph(Config *config) {
     return 0;
 }
 
-inline std::string query_sequence(size_t id,
-                                  const std::string &name,
-                                  const std::string &seq,
-                                  const AnnotatedDBG &anno_graph,
-                                  const Config &config) {
-    return QueryExecutor::execute_query(fmt::format_int(id).str() + '\t' + name, seq,
-                                        config.count_labels, config.print_signature,
-                                        config.suppress_unlabeled, config.num_top_labels,
-                                        config.discovery_fraction, config.anno_labels_delimiter,
-                                        anno_graph);
+inline void query_sequence(size_t id, std::string name, std::string seq,
+                           const AnnotatedDBG &anno_graph,
+                           const Config &config,
+                           const align::DBGAlignerConfig *aligner_config,
+                           const std::function<void(const std::string&)> &callback) {
+    if (aligner_config) {
+        auto matches = build_aligner(anno_graph.get_graph(), *aligner_config)->align(seq);
```

Comment:
should it be called `auto alignments = ...->align(seq)`?

Reply:
sure, I called it `matches` just to be shorter